app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -29,47 +29,73 @@
                     sort=this.sort
                 ) as |SortButton|}}
                     <tr>
-                        <th data-test-header-name rowspan='2'>
-                            <strong>{{t 'institutions.dashboard.users_list.name'}}</strong>
-                            <SortButton @sortBy='user_name' />
+                        <th data-test-header-name rowspan='3'>
+                            <div class="header-content">
+                                <strong>{{t 'institutions.dashboard.users_list.name'}}</strong>
+                                <SortButton @sortBy='user_name' />
+                            </div>
                         </th>
                         <th data-test-header-department rowspan='2'>
-                            <strong>{{t 'institutions.dashboard.users_list.department'}}</strong>
-                            <SortButton @sortBy='department' />
+                            <div class="header-content">
+                                <strong>{{t 'institutions.dashboard.users_list.department'}}</strong>
+                                <SortButton @sortBy='department' />
+                            </div>
                         </th>
-                        <th local-class='nested-header text-center' colspan='2' width='128'>
-                            {{t 'institutions.dashboard.users_list.projects'}}
+                        <th>
+                            {{t 'institutions.dashboard.users_list.osf_link'}}
                         </th>
-                    </tr>
-                    <tr>
-                        <th local-class='nested-header text-center' data-test-header-public-projects>
-                            <img src='/assets/images/global/open-lock.svg'
-                                alt={{t 'institutions.dashboard.users_list.public_projects'}}
-                            >
+                        <th>
+                            <div class="header-content">
+                                <span>{{t 'institutions.dashboard.users_list.public_projects'}}</span>
+                                <SortButton @sortBy='public_projects' />
+                            </div>
+                        </th>
+                        <th style="padding-left: 0px;" >
+                            <div class="header-content">
```

Comment:
We try to avoid using any inline-styling, so I would remove this `style="padding-left..."` and if needed, move it into a local-class.

`class="header-content"` can be removed safely as well